### PR TITLE
Document gapless mp3 encoding.

### LINKED ISSUE #1741: Convert MP3 default isn't gapless
When importing losless files, the `convert` plugin's default mp3 command-line discards length information:

```
ffmpeg -i $source -y -vn -aq 2 $dest
```

Instead, I believe that the following command-line should be default:

```
ffmpeg -i $source -f wav - | lame -V 2 --noreplaygain - $dest
```

This ensures that delay/padding information is preserved in the [LAME MP3 Info Tag](http://wiki.hydrogenaud.io/index.php?title=MP3#LAME_tag), enabling gapless playback on [many devices / applications](https://en.wikipedia.org/wiki/Gapless_playback#Player_support). The downside, of course, is that the `lame` executable is required in addition to `ffmpeg`.

Would you accept a pull-request to change the default command-line, or at least document this shortcoming of the default and the alternative gapless solution?

### PR DESCRIPTION
**Pull request by smlx**
Fixes #1741.

### CODE REVIEW

**Comment by Kraymer** [comment]
High quality .rst you produced here ! :shark:

File: docs/plugins/convert.rst [hunk 1]
```diff
@@ -140,3 +140,52 @@ and the given command is used for all conversions.
...
+.. _LAME: http://lame.sourceforge.net/
+
+To do this on Linux, you need to install ``lame``, create a script (e.g.
+``/home/user/.config/beets/mp3.sh``) with the following contents, and make it
+executable.
+
+::
+
+    #!/bin/sh
+    /usr/bin/ffmpeg -i "$1" -f wav - | /usr/bin/lame -V 2 --noreplaygain - "$2"
```
Comment (Kraymer) [inline docs/plugins/convert.rst]: Maybe don't put the absolute paths which may vary :

    ffmpeg -i "$1" -f wav - | lame -V 2 --noreplaygain - "$2"

Comment (smlx) [inline docs/plugins/convert.rst]: Sure, should be fixed now! 😄

**Comment by sampsyo** [comment]
Thank you for adding all this detail! I compressed things down a little bit for brevity and merged. :sparkles:

**Pull request merged**